### PR TITLE
Cloudera KB Articles Update

### PR DESCRIPTION
**Pull request by timjbaumgartner**
Updated the getting started with Cloudera KB article with up to date screenshots, current information on add-on components and added additional Q & A's. Added links to the VPN KB article and the Cloudera PDF in the attachments folder. 
Updated the PDF link in the Cloudera Manager Guide KB article.
Created a Cloudera folder in the images folder and added the six screenshots.

Thanks, Tim Baumgartner 
tim.baumgartner@centurylink.com

### CODE REVIEW

File: Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md [hunk 1]
```diff
@@ -10,35 +10,60 @@
 <p>Here’s how to create a managed Cloudera Hadoop environment in CenturyLink Cloud.</p>
 <p><strong>1.&nbsp;Search for “Cloudera” in the Blueprints library. Then, click the version and cluster configuration Blueprint that suits your needs.</strong>
 </p>
-<p><img src="https://t3n.zendesk.com/attachments/token/RWzHCc732wFk8dPiiGo1bnnsY/?name=Cloudera_Article_1.png" alt="Cloudera_Article_1.png" />
+<p><img src="../images/Cloudera/Cloudera_Article_1.PNG" alt="CLC Search Cloudera Blueprint" />
```
Comment (nathanyoung) [inline Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md]: broken image link, probably case sensitive file name.

File: Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md [hunk 2]
```diff
@@ -10,35 +10,60 @@
...
 </p>
 <p><strong>2. Click on the "Deploy Blueprint" button.</strong>
 </p>
-<p>&nbsp;<img src="https://t3n.zendesk.com/attachments/token/esjEXVwmqWjI7DHGSNtrAdGW1/?name=Cloudera_Article_2.png" alt="Cloudera_Article_2.png" /></p>
+<p><img src="../images/Cloudera/Cloudera_Article_2.PNG" alt="CLC Deploy Cloudera Blueprint" /></p>
```
Comment (nathanyoung) [inline Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md]: broken

File: Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md [hunk 3]
```diff
@@ -10,35 +10,60 @@
...
 <p><strong>3.&nbsp;Fill out the appropriate details for the CLC Managed Cloudera Hadoop Blueprint.</strong>
 </p>
-<p><img src="https://t3n.zendesk.com/attachments/token/Ob13N2J0Z800Hwp6MuXp0fDWt/?name=Cloudera_Article_3.png" alt="Cloudera_Article_3.png" />
+<p><img src="../images/Cloudera/Cloudera_Article_3.PNG" alt="CLC Build Cloudera Servers" />
```
Comment (nathanyoung) [inline Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md]: broken

File: Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md [hunk 4]
```diff
@@ -10,35 +10,60 @@
...
 </p>
-<p><strong>5.&nbsp;Once verified, click on the ‘deploy blueprint’ button. You will see the deployment details along with an email stating the Blueprint is queued for execution.</strong>
+<p><strong>5.&nbsp;Verify your configuration details.</strong>
 </p>
-<p><img src="https://t3n.zendesk.com/attachments/token/65hVJPzEIf7LYj6d8HFt4HygP/?name=Cloudera_Article_5.png" alt="Cloudera_Article_5.png" /></p>
+<p><img src="../images/Cloudera/Cloudera_Article_5.PNG" alt="CLC Verify Cloudera Configuration" />
+</p>
+<p><strong>6.&nbsp;Once verified, click on the ‘deploy blueprint’ button. You will see the deployment details along with an email stating the Blueprint is queued for execution.</strong>
+</p>
+<p><img src="../images/Cloudera/Cloudera_Article_6.PNG" alt="CLC Cloudera Blueprint Executing" /></p>
```
Comment (nathanyoung) [inline Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md]: broken

File: Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 {{{
   "title": "Getting Started with Cloudera on CenturyLink Cloud",
-  "date": "10-30-2014",
-  "author": "Jared Ruckle",
+  "date": "04-21-2015",
+  "author": "Tim Baumgartner",
   "attachments": [],
   "contentIsHTML": true
```
Comment (nathanyoung) [inline Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md]: should be `false` since you're adding markdown syntax links

**Comment by timjbaumgartner** [comment]
make changes based on comments

File: Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md [hunk 6]
```diff
@@ -10,35 +10,60 @@
...
+<ul>
+ <li>Accumulo</li>
+ <li>HBase</li>
+ <li>Impala</li>
+ <li>Key-Value Store Indexer (Requires HBase and Solr)</li>
+ <li>Navigator (Not available for the Express Version)</li>
+ <li>Sentry</li>
+ <li>Solr</li>
+ <li>Spark</li>
+<ul/>
```
Comment (nathanyoung) [inline Managed Services/getting-started-with-cloudera-on-centurylink-cloud.md]: should be `</ul>`

**Pull request merged**